sdks/python/test-suites/tox/py38/build.gradle

```diff
@@ -34,16 +34,11 @@ apply from: "../common.gradle"
 // TODO(BEAM-8954): Remove this once tox uses isolated builds.
 testPy38Cython.mustRunAfter testPython38, testPy38CloudCoverage
 
-toxTask "testPy38pyarrow-0", "py38-pyarrow-0"
-toxTask "testPy38pyarrow-1", "py38-pyarrow-1"
-toxTask "testPy38pyarrow-2", "py38-pyarrow-2"
-toxTask "testPy38pyarrow-3", "py38-pyarrow-3"
-toxTask "testPy38pyarrow-4", "py38-pyarrow-4"
-test.dependsOn "testPy38pyarrow-0"
-test.dependsOn "testPy38pyarrow-1"
-test.dependsOn "testPy38pyarrow-2"
-test.dependsOn "testPy38pyarrow-3"
-test.dependsOn "testPy38pyarrow-4"
+(0..5).each {version ->
```

Comment:
I don't see py38-pyarrow-5 in the logs: https://ci-beam.apache.org/job/beam_PreCommit_Python_Commit/19922/consoleFull#gradle-task-281

Syntax looks okay though: https://blog.mrhaki.com/2009/09/groovy-goodness-keep-your-values-in.html

Reply:
Weird. I do see pyarrow-5 locally:
```
❯ ./gradlew :sdks:python:test-suites:tox:py38:test --dry-run
...
:sdks:python:test-suites:tox:py38:testPy38pyarrow-0 SKIPPED
:sdks:python:test-suites:tox:py38:testPy38pyarrow-1 SKIPPED
:sdks:python:test-suites:tox:py38:testPy38pyarrow-2 SKIPPED
:sdks:python:test-suites:tox:py38:testPy38pyarrow-3 SKIPPED
:sdks:python:test-suites:tox:py38:testPy38pyarrow-4 SKIPPED
:sdks:python:test-suites:tox:py38:testPy38pyarrow-5 SKIPPED
:sdks:python:test-suites:tox:py38:test SKIPPED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.9.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 959ms
```

Reply:
Ah. I need to modify `sdks/python/test-suites/tox/common.gradle`: https://github.com/apache/beam/commit/20c48bb73c2b427b18e9448d6d6e839c90798fec#diff-0f6b7a4248348cee86b58c1b1ba8425f89ada266ee9bedb291f876f7fd0a61ed

Thanks for checking this :)

Reply:
Looks like it's running now. Unfortunately I noticed we've been running the pyarrow tests twice, once with xdist and once without - filed BEAM-12985 for this.